Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -711,6 +694,15 @@ extension TabTrayController: SettingsDelegate {
     }
 }
 
+extension TabTrayController: PhotonActionSheetProtocol {
+    func didTapMenu() {
+        let closeAllTabs = PhotonActionSheetItem(title: "Close All Tabs", iconString: "action_remove") { _ in
```

Comment:
Nit: use already localized `Strings.swift` string.

(There will already be a localized string, which if not in `Strings.swift` should be moved there).